src/commands/external.rs

```diff
@@ -66,7 +67,15 @@ pub async fn execute_external_subcommand(
     )
     .await?;
 
-    let mut command = Command::new(plugin_store.installed_binary_path(&plugin_name));
+    let binary = plugin_store.installed_binary_path(&plugin_name);
+    if !Path::new(binary.as_os_str()).exists() {
```

Comment:
`binary` is already a `PathBuf` so has the `exists` method built in - no need to extract the string or construct a `Path`.

Reply:
💯  thank you